### ISSUE TITLE
Windows Subsystem for Linux Support 

### ISSUE THREAD

**Issue by Celibioux** [TabbyML/tabby#1944]
**Please describe the feature you want**
Any plans to support Installation on WSL Please ? 

**Additional context**

---
Please reply with a 👍 if you want this feature.

**Comment by wsxiaoys** [comment]
WSL should work seamlessly with Docker support. We have also provided a Windows binary for native Windows support. Would these options be sufficient for your use case?"

**Comment by Celibioux** [comment]
I've Just discovered your software and i'm testing it out with your docker example but it's kinda slow with Docker and CPU only model that's why i was if WSL support would be better ?

**Comment by wsxiaoys** [comment]
Please refer https://tabby.tabbyml.com/docs/installation/windows/ for windows installation guide

**Comment by itstueben** [comment]
i install it on my windows machine with wsl2. I use the docker way and it looks very good. thx! The docker installation support my Nvidia card.

**Comment by juangcarmona** [comment]
For those interested on having tabby up and running on their windows, with WSL.

### **TL;DR;** 
Best way is to have do it is to setup an Ubuntu Distro, with WSL, and follow Linux installation instructions. 

I wrote an article with the details and some examples and tips --> **[A Fast and Private AI-Coding Assistant](https://jgcarmona.com/run-ai-locally-with-tabby-ml/)**. 

**How-To-Guide in 3 simple steps:**

1. Install Ubuntu (22.04 or 24.04, both worked for me)
2. Follow [Linux installation instructions](https://tabby.tabbyml.com/docs/quick-start/installation/linux/)
3. [OPTIONAL] You should also install katana so that you can index developer documentation. (I am about to create a PR with documentation improvements regarding exactly this), 

---

In addition, I have to say that I tried many Tabby flavors, and the best way to fully use my GPU is this approach. (Indeed I couldn't make it properly work on Windows and had a lot of difficulties with the container one... With this approach I totally own my On Device AI, which is what I was looking for when I first found Tabby ML.
